src/snowflake/connector/cursor.py

```diff
@@ -436,7 +440,7 @@ def interrupt_handler(*_):  # pragma: no cover
 
     def execute(self,
                 command: str,
-                params: Union[List, Tuple, None] = None,
+                params: Union[List, Tuple, None, str] = None,
```

Comment:
Split this into a private argument instead as per our conversation.
In that case will this work with `pyformat` too?

Reply:
I don't think so, `pyformat` always uses client side binding, I'm not sure if GS will parse the query correctly and do `COPY INTO` even if we upload bindings as files.